client.go

```diff
@@ -190,6 +190,26 @@ func (c *Client) trigger(channels []string, event string, data interface{}, sock
 	return unmarshalledBufferedEvents(response)
 }
 
+type BatchRequest struct {
```

Comment:
Is it worth making the `BatchRequest` public? None of the functions allow it as a parameter